### PR TITLE
test: Cypress - Flaky Fix

### PR DESCRIPTION
**Pull request by Vijetha-Kaja**
## Description

- Fixed previous PR review comments

**Fixed below flaky tests**

- TableV2_PropertyPane_1_spec.js
- Table_Derived_Column_Data_validation_spec.js
- ConversionFlow_Corner_Cases_spec.ts
- Topstacked_spec.ts
- FirstTimeUserOnboarding_spec.js

## Type of change

- Flaky test fix

## How Has This Been Tested?
- Cypress test runs

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/BugTests/Bug15056_Spec.ts [hunk 1]
```diff
@@ -12,10 +13,7 @@ describe("JS data update on button click", function () {
   });
 
   it("1. Populates js function data when triggered via button click", function () {
-    apiPage.CreateAndFillApi(
-      "https://jsonplaceholder.typicode.com/posts",
-      "Api1",
-    );
+    apiPage.CreateAndFillApi(testData.baseUrl + testData.methods, "Api1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/BugTests/Bug15056_Spec.ts:16]: Why not use mock TED url `mockApiUrl` directly? Why is testData needed here?

File: app/client/cypress/e2e/Regression/ClientSide/BugTests/GitBugs_Spec.ts [hunk 2]
```diff
@@ -15,7 +16,7 @@ describe("Git Bugs", function () {
   });
 
   it("1. Bug 16248, When GitSync modal is open, block shortcut action execution", function () {
-    const largeResponseApiUrl = "https://jsonplaceholder.typicode.com/users";
+    const largeResponseApiUrl = testData.baseUrl + testData.methods;
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/BugTests/GitBugs_Spec.ts:19]: Same as above!

File: app/client/cypress/e2e/Regression/ClientSide/Linting/BasicLint_spec.ts [hunk 3]
```diff
@@ -79,7 +80,7 @@ describe("Linting", () => {
     clickButtonAndAssertLintError(true);
 
     // create Api1
-    apiPage.CreateAndFillApi("https://jsonplaceholder.typicode.com/");
+    apiPage.CreateAndFillApi(testData.baseUrl + testData.methods);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Linting/BasicLint_spec.ts:83]: Same at all places in this PR.

File: app/client/cypress/e2e/Regression/ClientSide/MobileResponsiveTests/ConversionFlow_Corner_Cases_spec.ts [hunk 4]
```diff
@@ -68,7 +68,9 @@ describe("Handle Cases while conversion", () => {
     () => {
       entityExplorer.AddNewPage("Add page from template");
       agHelper.AssertElementVisible(template.templateDialogBox);
-      agHelper.GetNClick("//h1[text()='Marketing Dashboard']");
+      agHelper.GetNClick(
+        "//h1[text()='Marketing Dashboard']/parent::div//button[contains(@class, 't--fork-template')]",
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/MobileResponsiveTests/ConversionFlow_Corner_Cases_spec.ts:72]: Why direct locator here?

File: app/client/cypress/e2e/Regression/ClientSide/Onboarding/FirstTimeUserOnboarding_spec.js [hunk 5]
```diff
@@ -186,13 +186,12 @@ describe("FirstTimeUserOnboarding", function () {
...
 
   it("3. onboarding flow - should check directly opening widget pane", function () {
-    cy.get(OnboardingLocator.checklistDatasourceBtn).should("be.visible");
+    agHelper.AssertElementVisible(OnboardingLocator.checklistDatasourceBtn);
     agHelper.GetNClick(OnboardingLocator.introModalCloseBtn);
     entityExplorer.NavigateToSwitcher("Widgets");
-    cy.get(OnboardingLocator.widgetSidebar).should("be.visible");
-    cy.get(OnboardingLocator.dropTarget).should("be.visible");
+    agHelper.AssertElementVisible(OnboardingLocator.widgetSidebar);
+    agHelper.AssertElementVisible(OnboardingLocator.dropTarget);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Onboarding/FirstTimeUserOnboarding_spec.js:193]: Changing to TS helpers is fixing it? How???

Comment (Vijetha-Kaja) [reply app/client/cypress/e2e/Regression/ClientSide/Onboarding/FirstTimeUserOnboarding_spec.js:193]: Removing extra re-load is the fix, i've just changed above commands to ts just to reduce burden in converting to ts later on

Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/Onboarding/FirstTimeUserOnboarding_spec.js:193]: Ah, see that now! js was extra here!

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**